content/reference/enums.md

```diff
@@ -256,3 +256,36 @@ secondary_residence | Secondary residence.
 non_residential     | Non residential.
 {: class="table table-bordered"}
 
+## Inbox Message Channels
+
+Name             | Description
+-----------------|------------
+airbnb           | Airbnb
+homeaway         | HomeAway
+email            | Email
+{: class="table table-bordered"}
+
+## Inbox Message Origins
+
+Name              | Origin
+------------------|------------
+bookingsync_inbox | BookingSync Inbox
+airbnb_message    | Airbnb message
+airbnb_email      | Airbnb email
+booking-com       | booking.com
```

Comment:
Booking.com in the  origin should be capitalised, especially if it's one our more biggest partners :P